ai_genomics/analysis/ai_genomics_experts/cb_orgs_and_people.py

```diff
@@ -0,0 +1,143 @@
+        - Adds additional organisation related columns
+        - Creates `description` column which uses long_description
+            or short_description depending what is available
+        - Selects relevant columns
+        - Filters out closed organisations
+
+    Args:
+        ai_gen_cb_orgs: AI genomics Crunchbase organisations
+        cb_orgs: Crunchbase organisations (to get additional
+            columns from)
+
+    Returns:
+        Processed AI genomics Crunchbase organisations
+    """
+    return (
+        ai_gen_cb_orgs.merge(cb_orgs, how="left", left_on="cb_org_id", right_on="id")
+        .assign(description=lambda x: x.long_description.fillna(x.short_description))[
+            [
+                "cb_org_id",
+                "name",
+                "description",
+                "cb_url",
+                "homepage_url",
+                "total_funding_usd",
+                "country",
+                "roles",
+                "closed_on",
+                "employee_count",
+            ]
+        ]
+        .query("closed_on.isnull()")
+        .drop(columns="closed_on")
+    )
+
+
+def select_and_rename_cb_people_cols(cb_people: pd.DataFrame) -> pd.DataFrame:
+    """Select and rename columns for Crunchbase people dataframe"""
+    return cb_people[
+        [
+            "uuid",
+            "name",
+            "cb_url",
+            "featured_job_organization_uuid",
+            "featured_job_organization_name",
+            "featured_job_title",
+        ]
+    ].rename(
+        columns={
+            "uuid": "person_id",
+            "name": "person_name",
+            "featured_job_organization_uuid": "org_id",
+            "featured_job_organization_name": "org_name",
+            "featured_job_title": "job_title",
+        }
+    )
+
+
+def find_top_n_most_funded_ai_genomics_orgs(
+    ai_gen_cb_orgs: pd.DataFrame, n: int
+) -> pd.DataFrame:
+    """Find the top n most funded AI genomics Crunchbase organisations
+
+    Args:
+        ai_gen_cb_orgs: AI genomics Crunchbase organisations
+        n: Number of top companies to select
+
+    Returns:
+        Top n most funded AI genomics organisations
+    """
+    return (
+        ai_gen_cb_orgs.dropna(subset="total_funding_usd")
+        .astype({"total_funding_usd": float})
+        .sort_values(by=["total_funding_usd"], ascending=False)
+        .query("roles == 'company'")
+        .head(n)
+        .reset_index(drop=True)
+    )
+
+
+def find_people_related_to_orgs(
+    cb_people: pd.DataFrame, cb_org_ids: list
+) -> pd.DataFrame:
+    """Find Crunchbase people related to specified cb_org_ids
+
+    Args:
+        cb_people: Crunchbase people dataframe
+        cb_org_ids: List of Crunchbase org ids
+
+    Returns:
+        Crunchbase people related to specified cb_org_ids
+    """
+    return cb_people.query(f"org_id in {cb_org_ids}").reset_index(drop=True)
+
+
+if __name__ == "__main__":
```

Comment:
I checked the outputs and looks like there isn't the same problem as there is with patent assignees - looks good!